config.js

```diff
@@ -27,10 +27,9 @@ module.exports = {
     host: process.env.PGHOST,
     port: process.env.PGPORT,
     database: process.env.PGNAME || process.env.DB_NAME || NAME,
-    user: process.env.PGUSER,
-    password: process.env.PGPASS,
+    user: process.env.PGUSER || 'postgres',
```

Comment:
Good catch. Looks like this is reflected in the docs. @kadamwhite @isaacdurazo @mzgoddard et al, if you are using mac osx and running postgres natively, make sure to adjust your system according.